### PR TITLE
MAT-6044 update VSAC connection

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [x] Tests have been run locally and pass.

## Screenshots (if appropriate)

- [ ] None applicable

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     </parent>
     <groupId>CMAT</groupId>
     <artifactId>MeasureAuthoringTool</artifactId>
-    <version>6.11</version>
+    <version>6.12</version>
```
Comment (jkotanchik-SB) [inline pom.xml]: Unless I'm overruled, I'll push for retaining the 6.11 version since the output of the tool is unchanged.

**Review by jkotanchik-SB — CHANGES_REQUESTED** [review]
Revert MAT version value.

File: src/main/java/mat/vsac/VsacService.java [hunk 2]
```diff
@@ -699,4 +570,15 @@ private HttpEntity<MultiValueMap<String,String>> buildEntityWithUrlEncodedBody(M
         headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
         return new HttpEntity<>(body, headers);
     }
+    
+    private HttpEntity<String> getHeaderEntityWithAuthentication(String apiKey) {
+    	HttpHeaders headers = new HttpHeaders();
+      headers.add("Authorization", getBasicAuthenticationHeader("apikey", apiKey));
```
Comment (jkotanchik-SB) [inline src/main/java/mat/vsac/VsacService.java]: Spring has a shortcut method for setting Basic Auth:
```suggestion
      headers.setBasicAuth("apikey", apiKey);
```

**Review by jkotanchik-SB — APPROVED** [review]
